config.yaml

```diff
@@ -1,7 +0,0 @@
-# Copyright 2022 Canonical Ltd.
-# See LICENSE file for licensing details.
-
-options:
-  extra-packages:
-    description: "Additional packages to install through APT"
-    type: "string"
```

Comment:
Keeping this would mean keeping the apt library as well and manually installed packages may not interact well with the snapped applications